### ISSUE TITLE
Compatibility with `discretize>=0.6.0`

### ISSUE THREAD

**Issue by prisae** [empymod/emg3d#147]
**EDIT**

This issue has two steps:
- [x] Make it compatible with `discretize v6.0.0` => #148 ; this is a bug, as it will currently fail;
- [x] Make it compatible for a future version `discretize v1.0.0` => currently I count that as an enhancement.

The second part requires to implement all the name changes.

---

There need to be some fixes to make emg3d compatible with discretize>=0.6.0. Most things should work, but there are some tests that fail. Currently it is fixed in:

- .travis.yml: `'discretize<0.6.0'`
- requirements-dev.txt: `discretize<0.6.0`
- runtests.sh: `'discretize<0.6.0'`

=> Fix this and undo the above three lines.

**Comment by prisae** [comment]
For a start, this is no longer possible (in `meshes.py`):
```
    import discretize.TensorMesh as dTensorMesh
```
and it has to be replaced by:
```
    from discretize import TensorMesh as dTensorMesh
```

But then there are at least 12 tests failing due to changes in `discretize`.

**Comment by prisae** [comment]
It is all fixed now in #148.

We still have to implement all the name changes that came with `discretize v0.6.0` - as some will result in Errors down the road. But this is not as urgent as pure compatibility with v0.6+

**Comment by prisae** [comment]
For reference for the relevant name changes:
http://discretize.simpeg.xyz/en/master/release/0.6.0-notes.html#changes

**Comment by prisae** [comment]
Part I solved. Re-opening for part II, changing from bug to maintenance.

**Comment by prisae** [comment]
There seem to be more issues with part 1 => #149 

Re-run gallery to check compatibility.

**Comment by prisae** [comment]
#153

**Comment by prisae** [comment]
Second part also fixed, in #153
